### ISSUE TITLE
ios simulator crashed when calling setDefaultAudioRoutetoSpeakerphone(true) in v.3.1.2

### ISSUE THREAD

**Issue by aitaro** [AgoraIO/Flutter-SDK#198]
**Describe the bug**
ios simulator crashed when calling setDefaultAudioRoutetoSpeakerphone(true)
Output log:

```
Fatal error: Unexpectedly found nil while unwrapping an Optional value: file agora_rtc_engine/RtcEngine.swift, line 708
```

Is it a bug, or I'm misunderstanding something?

**To Reproduce**
Problem part of my source code.
```
  // Platform messages are asynchronous, so we initialize in an async method.
  Future<void> initPlatformState() async {
    await [Permission.camera, Permission.microphone, Permission.storage]
        .request();

    var engine = await RtcEngine.create('YOUR_APP_ID');
    await engine.enableVideo();
    await engine.setDefaultAudioRoutetoSpeakerphone(true);
    await engine.joinChannel(null, '123', null, 0);
  }
```

My forked repository.
https://github.com/aitaro/Flutter-SDK/blob/master/example/lib/main.dart

**Smartphone (please complete the following information):**
iOS simulator
 - Device: iPhoneXI
 - OS: iOS14

**Comment by hanleiZoro** [comment]
me too.

**Comment by LichKing-2234** [comment]
have you set the correct AppID?

**Comment by aitaro** [comment]
Yes I have. I can use other features of AgoraSDK.

**Comment by LichKing-2234** [comment]
fix the next version, you can modify code locally before published.
![image](https://user-images.githubusercontent.com/9312526/95723939-4fee7a00-0ca8-11eb-969d-f45e431393f2.png)

**Comment by LichKing-2234** [comment]
fixed in 3.1.3
